### DISCUSSION TITLE
How can I remove persisted data in storage?

### DISCUSSION THREAD

**Discussion by pheng0077** [iendeavor/pinia-plugin-persistedstate-2#31] [Q&A]
I've tried to update and remove persisted data in the storage by setting the state to null, but the data in storage still exist.
I'm using Nuxt 3.

### In plugin/piniapersistedstate.ts
`export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.$pinia.use(
    createPersistedStatePlugin({
      serialize: (state) => stringify(state),
      deserialize: (state) => JSON.parse(state),
      storage: {
        getItem: async (key) => {
          return localforage.getItem(key);
        },
        setItem: async (key, value) => {
          return localforage.setItem(key, value);
        },
        removeItem: async (key) => {
          return localforage.removeItem(key);
        },
      },
    })
  );
});`

### In store/user.ts
`export const user = defineStore("user", () => {
  const credentials = ref<any | null>(null);

  const setUser = (value: any) => {
    credentials.value = value;
  };

  const removeUser = () => {
    credentials.value = null;
  };
  return { credentials, setUser, removeUser };
});`

**Comment by soc221b** [comment]
What is the `stringify` function you use in `serialize` option?

Could you provide a minimal reproduce?
